main.go

```diff
@@ -131,7 +141,13 @@ func fetchReleases(client *helm.Client) ([]*release.Release, error) {
 
 func fetchIndices(client *helm.Client) ([]*repo.IndexFile, error) {
 	indices := []*repo.IndexFile{}
-	rfp := os.Getenv("HELM_PATH_REPOSITORY_FILE")
+	rfp, exists := os.LookupEnv("HELM_PATH_REPOSITORY_FILE")
+	if !exists {
+		usr, err := user.Current()
```

Comment:
user.Current only works on linux IIRC. It might be better to use `k8s.io/client-go/util/homedir` and `homedir.HomeDir()` as it supports alll major distros.

Reply:
My understanding was that user.Current also works on windows I can change it to use that if you prefer.